### ISSUE TITLE
Allow deletion of multiple not adjusted cells in a DataSource bound Spreadsheet

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#3903]
### Bug report
Currently, DataSource bound Spreadsheet throws "Unsupported for multiple ranges" error when attempting to delete the content from multiple not adjusted cells.

The error is thrown by the *values()* method:
```JavaScript
values: function (values) {
    if (this._ref instanceof UnionRef) {
        throw new Error('Unsupported for multiple ranges.');
    }
    if (this._ref === kendo.spreadsheet.NULLREF) {
....
```

### Reproduction of the problem
* Go to the [DataSource Binding demo](https://demos.telerik.com/kendo-ui/spreadsheet/datasource);
* Select a few cells using the Ctrl + click;
* Press *Delete* button on the keyboard;
* "Unsupported for multiple ranges" error will be thrown from the *values()* method;

**Comment by sqlnew** [comment]
Yes, I also met this bug, hoping to repair it as soon as possible.

**Comment by sqlnew** [comment]
This bug has not been resolved in Version 2018.3.911. Which mail can I send to reflect this problem?

**Comment by sqlnew** [comment]
When can we solve this problem? I hope telerik will pay attention to this problem.

**Comment by sqlnew** [comment]
IIII, Unfortunately, this Bug has not been repaired in the v2019.1.115 version. It's disappointing.
When can we solve this problem? I hope telerik will pay attention to this problem.???
